package/MDAnalysis/core/topologyattrs.py

```diff
@@ -885,7 +885,7 @@ def principal_axes(group, pbc=None):
         e_val, e_vec = np.linalg.eig(atomgroup.moment_of_inertia(pbc=pbc))
 
         # Sort
-        indices = np.argsort(e_val)
+        indices = np.argsort(e_val[::-1])
```

Comment:
Reverse the indices, not the e_val because according to the docs for `linalg.eig`, **The eigenvalues are not necessarily ordered.**
```python
indices = np.argsort(e_val)[::-1]
```